CommonRegexes.h

```diff
@@ -0,0 +1,28 @@
+#ifndef COMMON_REGEXES_H
+#define COMMON_REGEXES_H
+
+
+
+namespace commonItems
+{
+
+// catchall:
+//		We grab everything that's NOT =,  =  or , OR we grab everything within quotes, except newlines, which we already
```

Comment:
This comment may be corrupted, in the ParserHelpers it was "[...]NOT =, { or }, OR we grab everything[...]".